flavors/anexa/web/css/main.css

```diff
@@ -29,7 +29,7 @@ a img
     height:auto;
     margin:0;
     padding:0;
-    background: #549876;
+    background-color: #172983;
```

Comment:
No se por que esta así. Mañana lo reviso, porque no recuerdo.

-----Original Message-----
From: María Emilia Corrons <notifications@github.com>
Date: Tue, 31 May 2016 12:23:46 
To: Desarrollo-CeSPI/kimkelen<kimkelen@noreply.github.com>
Reply-To: Desarrollo-CeSPI/kimkelen <reply@reply.github.com>
Cc: nvidela<nai.videla@gmail.com>; Author<author@noreply.github.com>
Subject: Re: [Desarrollo-CeSPI/kimkelen] Cambio estado alumnos (#234)

> @@ -29,7 +29,7 @@ a img
>      height:auto;
>      margin:0;
>      padding:0;
> -    background: #549876;
> +    background-color: #172983;

No sé por qué estos cambios, el otro día mergee y no le presté atención a los estilos estos. pero cambiaron todos los colores de los headers

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/Desarrollo-CeSPI/kimkelen/pull/234/files/5a3e926da03d64f25361469458826f8be109834b#r65246503